response/core/urls.py

```diff
@@ -5,7 +5,7 @@
 
 # Routers provide an easy way of automatically determining the URL conf.
 router = routers.DefaultRouter()
-router.register(r'incidents', IncidentViewSet, base_name='Incidents')
+router.register(r'incidents', IncidentViewSet, basename='incident')
```

Comment:
Changed for consistency with the auto-generated base names for the other routes. `base_name` is also deprecated in favour of `basename`